fortranformat/_output.py

```diff
@@ -94,66 +97,14 @@ def output(eds, reversion_eds, values):
                     val = 0
                 sub_string = _compose_i_string(
                     ed.width, ed.min_digits, state, val)
-            elif isinstance(ed, B):
-                if val is None:
-                    val = 0
-                w = ed.width
-                m = ed.min_digits
-                sub_string = _compose_boz_string(w, m, state, val, 'B')
-            if isinstance(ed, O):
-                if val is None:
-                    val = 0
-                w = ed.width
-                m = ed.min_digits
-                sub_string = _compose_boz_string(w, m, state, val, 'O')
-            if isinstance(ed, Z):
+            elif isinstance(ed, (B, O, Z)):

```

Comment:
I guess this could also be chaged for `in` if `isinstance` is slow i.e. `elif ed.name in ['B', 'O', 'Z']:`